test/MUXControlsTestApp/TestInventory.cs

```diff
@@ -12,48 +12,53 @@ static TestInventory()
-            Tests.Add(new TestDeclaration("RatingControl Tests", typeof(RatingControlPage)));
-            Tests.Add(new TestDeclaration("SwipeControl Tests", typeof(SwipePage)));
-            Tests.Add(new TestDeclaration("TreeView Tests", typeof(TreeViewPage)));
-            Tests.Add(new TestDeclaration("TeachingTip Tests", typeof(TeachingTipPage)));
-            Tests.Add(new TestDeclaration("TwoPaneView Tests", typeof(TwoPaneViewPage)));
-            Tests.Add(new TestDeclaration("PTR Tests", typeof(PTRPage)));
-            Tests.Add(new TestDeclaration("MenuBar Tests", typeof(MenuBarPage)));
-            Tests.Add(new TestDeclaration("MenuFlyout Tests", typeof(MenuFlyoutPage)));
-            Tests.Add(new TestDeclaration("ScrollViewerAdapter Tests", typeof(ScrollViewerAdapterPage)));
-            Tests.Add(new TestDeclaration("SplitButton Tests", typeof(SplitButtonPage)));
-            Tests.Add(new TestDeclaration("DropDownButton Tests", typeof(DropDownButtonPage)));
-            Tests.Add(new TestDeclaration("CommandBarFlyout Tests", typeof(CommandBarFlyoutMainPage)));
-            Tests.Add(new TestDeclaration("CommonStyles Tests", typeof(CommonStylesPage)));
-            Tests.Add(new TestDeclaration("Compact Tests", typeof(CompactPage)));
-            Tests.Add(new TestDeclaration("RadioButtons Tests", typeof(RadioButtonsPage)));
-            Tests.Add(new TestDeclaration("RadioMenuFlyoutItem Tests", typeof(RadioMenuFlyoutItemPage)));
+            Tests.Add(new TestDeclaration("ColorPicker", typeof(ColorPickerPage), "ms-appx:///Assets/ColorPicker.png"));
+            Tests.Add(new TestDeclaration("Leak", typeof(LeakTestPage)));
+            Tests.Add(new TestDeclaration("PersonPicture", typeof(PersonPicturePage), "ms-appx:///Assets/PersonPicture.png"));
+            Tests.Add(new TestDeclaration("RatingControl", typeof(RatingControlPage), "ms-appx:///Assets/RatingControl.png"));
+            Tests.Add(new TestDeclaration("SwipeControl", typeof(SwipePage), "ms-appx:///Assets/Swipe.png"));
+            Tests.Add(new TestDeclaration("TreeView", typeof(TreeViewPage), "ms-appx:///Assets/TreeView.png"));
+            Tests.Add(new TestDeclaration("TeachingTip", typeof(TeachingTipPage), "ms-appx:///Assets/TeachingTip.png"));
+            Tests.Add(new TestDeclaration("TwoPaneView", typeof(TwoPaneViewPage)));
+            Tests.Add(new TestDeclaration("PTR", typeof(PTRPage), "ms-appx:///Assets/PullToRefresh.png"));
+            Tests.Add(new TestDeclaration("MenuBar", typeof(MenuBarPage), "ms-appx:///Assets/MenuBar.png"));
+            Tests.Add(new TestDeclaration("MenuFlyout", typeof(MenuFlyoutPage), "ms-appx:///Assets/MenuFlyout.png"));
+            Tests.Add(new TestDeclaration("ScrollViewerAdapter", typeof(ScrollViewerAdapterPage), "ms-appx:///Assets/ScrollViewer.png"));
+            Tests.Add(new TestDeclaration("SplitButton", typeof(SplitButtonPage), "ms-appx:///Assets/SplitButton.png"));
```

Comment:
>ms-appx:///Assets/ [](start = 83, length = 18)

To be short to "SplitButton.png", we don't need to add ms-appx:///Asserts everywhere.